src/Infrastructure/KafkaJanitor.RestClient/Features/Topics/TopicsClient.cs

```diff
@@ -34,10 +35,16 @@ public async Task CreateAsync(Topic input)
             );
         }
 
-        public async Task<IEnumerable<Topic>> GetAllAsync()
+        public async Task<IEnumerable<Topic>> GetAllAsync(string clusterId)
```

Comment:
In general it would be a good idea to implement requests as stand-alone objects to abide by the SRP principles (if we must)